### PR TITLE
fix: discard polygons with length of 4 or less

### PR DESCRIPTION
**Pull request by savan77**
It was found that Matterport's MaskRCNN implementation generates polygons with length of 4 which creates lot of issues since pycocotools considers them as a bounding box. It looks like these polygons with length of 4 are just garbage values. That is actually a single point repeated twice. So, we can just filter them out.
More discussion around this topic can be found here #1769.

### Motivation and context

It will solve issues where users were not able to see the job because there were polygons with length of 4. It will also solve an issue with export dataset feature as we go through pycocotools there as well.

### How has this been tested?

This has been tested locally as well as on Onepanel.

### Checklist

- [`x`] I submit my changes into the `develop` branch
- [`x` ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [`x` ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: serverless/tensorflow/matterport/mask_rcnn/nuclio/model_loader.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def infer(self, image, threshold):
                 contour = np.flip(contour, axis=1)
                 # Approximate the contour and reduce the number of points
                 contour = approximate_polygon(contour, tolerance=2.5)
-                if len(contour) < 3:
+                if len(contour) < 5:
```
Comment (zhiltsov-max) [inline serverless/tensorflow/matterport/mask_rcnn/nuclio/model_loader.py:66]: It might be better to use 6 here, though.

**Review by zhiltsov-max — APPROVED** [review]

File: serverless/tensorflow/matterport/mask_rcnn/nuclio/model_loader.py (hunk 1 above)
Comment (savan77) [reply serverless/tensorflow/matterport/mask_rcnn/nuclio/model_loader.py]: Agreed. Updated.

**Comment by nmanovic** [comment]
@savan77 , need to add a line into CHANGELOG. Also please check `I submit my code changes under the same MIT License`.  Probably server itself should have corresponding checks and don't accept "bad" polygons.

**Comment by savan77** [comment]
@savan77 Made changes as per your suggestions. Yeah, that might one possible way. Let me know what do you guys think. For now, this will solve multiple errors (https://github.com/opencv/cvat/issues/1769) that arises because of MaskRCNN generating polygons with two points (both are same).

**Review by nmanovic — APPROVED** [review]

**Review by nmanovic — APPROVED** [review]

**Review by nmanovic — APPROVED** [review]

**Pull request merged**